utils.js

```diff
@@ -673,7 +673,14 @@ const createCloudfrontDistribution = async (cf, subdomain, certificateArn, regio
 /*
  * Updates a distribution's origins
  */
-const updateCloudfrontDistribution = async (cf, subdomain, distributionId, region, triggers) => {
+const updateCloudfrontDistribution = async (
```

Comment:
🤔 Not related to this PR, but I was wondering if this package is the "only" way to do handle the domain/sls, or rather: what would be the appropriate way to do this if we were to implement this today?

It seems to be something that should live in a yaml serverless template, terraform or in the aula-cli.

Reply:
Hmmm, technically, this is still used in a `serverless.yaml` file. However I do agree that is weird!

We are essentially using `serverless` as a runtime to ensure existance of an S3 bucket, contents of the bucket (i.e. the built frontend app), the CDN configuration (CloudFront) and Route53. 

These steps, on their own, have nothing to do to `serverless` or `lambdas`, as the [serverless.tpl.yaml](https://github.com/AulaEducation/aula/blob/master/front/admin-dashboard/serverless.tpl.yaml#L3) file looks nothing like a standard `serverless` file (nor it's `serverless` as a concept).

My thinking, if I had to design a tooling mechanism for packaging a frontend app and deploying it to a CDN would be:
1. Have the infrastructure configuration close to the code, either by having a CloudFormation file, or even maybe a Terraform plan. This would be used to ensure the creation of the needed AWS resources (S3 bucket, domain, CDN configuration). This would also guarantee that we can apply the CloudFormation or Terraform on every CI/CD run, to ensure that the supporting infrastructure is in the right state
2. Build the app and upload the contents to the managed S3 bucket (this could be done with a simple `aws s3 cp` or even a [CircleCI Orb](https://circleci.com/docs/2.0/deployment-examples/)
3. Trigger a CloudFront invalidation (ensuring that clients are fetching the latest build), either with `aws cloudfront create-invalidation` or by using a [CircleCI Orb](https://circleci.com/developer/orbs/orb/topmonks/aws-cloudfront).